data/human/free worlds 2 middle.txt

```diff
@@ -1145,7 +1145,7 @@ mission "FW Northern 4.2A"
 mission "FW Alphas 1.1"
 	landing
 	name "Liberate Poisonwood"
-	description "Destroy all pirates ships in the <system> system with the help of the Navy Oathkeepers, then land on the planet and let the Navy troops eliminate the Alphas."
+	description "Destroy the pirate ships occupying the <system> system with the help of the Navy Oathkeepers, then land on the planet and let the Navy troops eliminate the Alphas."
```

Comment:
```suggestion
	description "Destroy the pirate ships occupying the <system> system with the help of the Navy Oathkeepers, then land on the planet and let the Navy troops eliminate the Alphas."
```

I agree this is an improvement while we're already fixing "all pirates ships," but just to double check:

Is the extra length going to cause space issues? Some of these mission descriptions are already pretty tight.

Reply:
It fits the description box.